### DISCUSSION TITLE
Columns in the output of "ps aux" are not aligned

### DISCUSSION THREAD

**Discussion by Jonson-zhang** [kingToolbox/WindTerm#1430] [Show and tell]
![image](https://user-images.githubusercontent.com/36395108/210150227-cae16379-d356-414c-9756-0db1c0e755b8.png)

**Comment by kingToolbox** [comment]
Sorry for the delay in replying to you. This bug has been fixed and will be released in version `WindTerm 2.6.0 Prerelease 3`. Please download and try it at your convenience. Thank you very much.
